app/src/main/res/layout/activity_selected_image_detail.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    >
+
+    <data>
+        <import type="android.view.View"/>
+        <variable
+            name="viewModel"
+            type="com.framgia.englishconversation.screen.selectedImageDetail.SelectedImageDetailViewModel"
+            />
+
+    </data>
+
+    <android.support.constraint.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context="com.framgia.englishconversation.screen.selectedImageDetail.SelectedImageDetailActivity"
+        >
+        <com.framgia.englishconversation.widget.ZoomableImageView
+            android:id="@+id/image_photo"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_margin="@dimen/dp_8"
+            android:adjustViewBounds="true"
+            android:scaleType="centerInside"
+            app:imageUrlFullSize="@{viewModel.mediaModels.get(viewModel.position).url}"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            />
+
+        <ImageButton
+            android:id="@+id/button_next"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/dp_8"
+            android:layout_marginEnd="@dimen/dp_8"
+            android:layout_marginRight="@dimen/dp_8"
+            android:layout_marginTop="8dp"
+            android:background="@null"
+            android:onClick="@{()->viewModel.nextImage()}"
+            android:src="@drawable/ic_right_arrow"
+            android:visibility="@{viewModel.position &lt; viewModel.mediaModels.size() -1 ? View.VISIBLE: View.GONE}"
+            app:layout_constraintBottom_toBottomOf="@+id/image_photo"
+            app:layout_constraintRight_toRightOf="@+id/image_photo"
+            app:layout_constraintTop_toTopOf="parent"
+            />
+        <ImageButton
+            android:id="@+id/button_back"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/dp_8"
+            android:layout_marginLeft="@dimen/dp_8"
+            android:layout_marginStart="@dimen/dp_8"
+            android:layout_marginTop="8dp"
+            android:background="@null"
+            android:onClick="@{()->viewModel.prevImage()}"
+            android:src="@drawable/ic_left_arrow"
```

Comment:
ERROR: [VectorDrawableCompat] When using VectorDrawableCompat, you need to use `app:srcCompat`.
 To use VectorDrawableCompat, you need to make two modifications to your project. First, set `android.defaultConfig.vectorDrawables.useSupportLibrary = true` in your `build.gradle` file, and second, use `app:srcCompat` instead of `android:src` to refer to vector drawables.